source/common/res/features/split-keyboard-shortcut/main.js

```diff
@@ -5,29 +5,54 @@
     ynabToolKit.splitKeyboardShortcut = (function () {
 
       return {
-        invoke: function () {
-        },
-
         observe: function (changedNodes) {
+          if (/accounts/.test(window.location.href)) {
+            if (changedNodes.regex(/^(?=.*\smodal-account-categories\s)(?=.*?\sactive)((?!closing).)*$/gm)) {
+              var splitButton = $(".button.button-primary.modal-account-categories-split-transaction");
+
+              // return if we are already inside a split subtransaction
+              if (splitButton.length < 1) return false;
+
+              var splitIcon = splitButton.html();
+              var categoryList = $(".modal-account-categories .modal-list");
+              var liElement = categoryList.find("li.user-data").last().clone();
 
-          // limit to accounts view
-          if (/accounts/.test(window.location.href))
-          {
-            if (changedNodes.has('modal-list'))
-            {
-              if ($('.ynab-grid-cell-subCategoryName input.accounts-text-field').val() == 'split')
-            {
-                $('button.modal-account-categories-split-transaction').mousedown();
-              }
+              liElement.find(".modal-account-categories-category").attr("title", "Split Transaction");
+              liElement.find(".modal-account-categories-category-name").html(splitIcon);
```

Comment:
There may be a problem with using JQuery's `.html()` in Firefox review, but I'm not sure how they'll feel when it's an element that's already on the page. They're hyper-sensitive to XSS and similar vulnerabilities, so they tend to get worried when they see `.html()`.

I'd say let's leave it and we'll have a look if we need to change it.